src/style.css

```diff
@@ -1,3 +1,96 @@
+:root {
+  --blue-color: rgba(0, 172, 220, 1);
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+  font-family: Roboto, serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you can use 'Roboto', Arial, sans-serif.